### DISCUSSION TITLE
Can't contract tags/folders on mobile

### DISCUSSION THREAD

**Discussion by alexgutts** [standardnotes/mobile#567] [Q&A]
Hi!
I have several folders on SN, i would love to see the functionality to contract my folders on sn, it would allow me to see way more space on my notes space, is is there a functionality for this coming in the future for mobile?

If not, is there a way I can suggest it?

**Comment by atmoio** [comment]
Yup we plan on improving the folders experience more on mobile over time.

**Reply by alexgutts** [reply]
> Yup we plan on improving the folders experience more on mobile over time.

We really need this feature, is there a way also to create tags on mobile?
